buildenv/jenkins/common/pipeline-functions

```diff
@@ -20,7 +20,7 @@
  * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
  *******************************************************************************/
 
- def get_shas(OPENJDK_REPO, OPENJDK_BRANCH, OPENJ9_REPO, OPENJ9_BRANCH, OMR_REPO, OMR_BRANCH) {
+ def get_shas(OPENJDK_REPO, OPENJDK_BRANCH, OPENJ9_REPO, OPENJ9_BRANCH, OMR_REPO, OMR_BRANCH, VENDOR_TEST_REPOS=null, VENDOR_TEST_BRANCHES=null, VENDOR_TEST_SHAS=null) {
```

Comment:
I assume having the `=null` added allows the parameters to be optionally passed? This function is currently only used by Pipeline-build-Test-Any-Platform so it seems like its adding complexity for no current gain. I see it will also be used by #2160 but that too could also pass the VENDOR_TEST* stuff.

Reply:
VENDOR_TEST* are optional because the function is also used in all of the Pipeline-Build-Test-JDK\<version\>-\<spec\> scripts.

Reply:
Those pipelines are no longer in use and are removed via #2467